src/agecalc.rs

```diff
@@ -0,0 +1,45 @@
+use std::fs::File;
+use std::io::Write;
+use std::io::{BufReader, BufRead};
+
+struct Person{
+    fname : String,
+    lname : String,
+    dob : String
+}
+
```

Comment:
Give function description in comments.....like give a descriptive comment on what is this function for?